packages/frontend-2/components/workspace/wizard/step/Details.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+  <WorkspaceWizardStep
+    title="Create a workspace"
+    description="Fill in some details for your teammates."
+  >
+    <form class="flex flex-col gap-4 w-full md:w-96" @submit="onSubmit">
+      <FormTextInput
+        id="workspace-name"
+        v-model:model-value="state.name"
+        name="name"
+        label="Workspace name"
+        color="foundation"
+        :rules="[isRequired, isStringOfLength({ maxLength: 512 })]"
+        show-label
+        auto-focus
+        size="lg"
+        @update:model-value="updateShortId"
+      />
+      <FormTextInput
```

Comment:
This was maybe already there, but there is a hydration mismatch on this:
<img width="448" alt="image" src="https://github.com/user-attachments/assets/7addc975-51a2-4d30-9c91-8687b36b12b5">